src/sql/workbench/services/notebook/browser/models/cell.ts

```diff
@@ -116,6 +117,11 @@ export class CellModel extends Disposable implements ICellModel {
 		}
 		// if the fromJson() method was already called and _cellGuid was previously set, don't generate another UUID unnecessarily
 		this._cellGuid = this._cellGuid || generateUuid();
+		if (this._cellType === 'code') {
+			this.cellLabel = localize('codeCellLabel', "Code Cell {0}", this.id);
```

Comment:
I don't think ID is appropriate here since that's just a random number. Generally for lists of objects like this you would announce the index and then the label (which would just be "Code Cell" or "Markdown Cell"). But getting this working correctly is likely out of scope for this PR so I guess this is fine - just putting this here for future reference. 

Although for code cells we should probably be saying the language as well.